Godeps/Godeps.json

```diff
@@ -12,7 +12,7 @@
 		{
 			"ImportPath": "bitbucket.org/ww/goautoneg",
 			"Comment": "null-5",
-			"Rev": "75cd24fc2f2c2a2088577d12123ddee5f54e0675"
+			"Rev": "'75cd24fc2f2c2a2088577d12123ddee5f54e0675'"
```

Comment:
couldn't say, it showed up in my, brendan's and huamin's update. I'm clueless why, but it works.